less/index.less

```
@@ -54,6 +54,127 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+
+	font-family: Arial, Helvetica, sans-serif;
+	// src: url(/fonts/Rubik-Regular.ttf);
+}
+
+h2 {
+	font-size: 24px;
+}
+
+@borderred: 1px solid red;
+@borderblue: 1px solid blue;
+@containercolor: #DCDCDC; 
+
+.flexboxrow {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-around;
+}
+
+.flexboxcolumn {
+	display: flex;
+	flex-direction: column;
+	margin: 0 auto; 
+	align-content: center;
 }
 
+.container {
+	max-width: 1000px;
+	background-color: @containercolor;
+	.flexboxcolumn();
+	@media (max-width: 500px) 
+	{
+
+	}
+}
+
+.section {
+	
+	// border: @borderred;
+	margin: 1%;
+	overflow: hidden;
+	
+	.flexboxrow(); 
+
+	.top-header {
+		width: 100%;
+		border: @borderblue; 
+		text-align: center; 
+		padding: 2%;
+	}
+
+	.header-image {
+		align-self: center;
+		width: 75%;
+	}
+
+	.item {
+		.flexboxcolumn();
+
+		padding: 3%;
+		border: @borderblue; 
+		margin: 2% 0%;
+		width: 33%;
+		overflow: hidden;
+		line-height: 125%;
+
+			.item-image 
+			{
+				text-align: center;
+				margin: 0 auto;
+				padding: 5%;
+
+					img	{
+						width: 75%;
+					}
+					
+			}
+
+			.item-text {
+				text-align: center;
+			}
+
+	}
+
+	.mainmenu {
+		align-self: center;
+		width: 80%;
+
+		nav {
+
+			ul {
+				.flexboxrow();
+				
+				li {
+					list-style-type: none;
+					margin: 1%;
+					padding: 2%;
+					
+					a {
+						border: @borderred;
+						text-decoration: none;
+						padding: 15%;
```

```suggestion
						padding: 15%;
```

i think this padding may be messing up the a tags when the view port changes.

maybe try 
```css
padding: 10px 5%
```

---

I got a chance to take a look at this this evening - still having the same issues, after adding the padding! Would love to hop on a zoom call later this coming afternoon to clarify.